package.json

```diff
@@ -57,5 +59,13 @@
   },
   "bugs": {
     "url": "https://github.com/reobin/vimcolorschemes/issues"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,json,md,css,scss}": "npm run format"
```

Comment:
It was my idea, but it turns out using `npm run format` over `prettier --write` makes the commit hook like 10x longer.

I didn't you could specify the extensions of the files you want formatted with husky, which is the only purpose of the `npm run format` command.

If you could change `npm run format` to `prettier --write`, then everything would be perfect.

Thanks for your work! 🎉 This will help a bunch with the PRs!

Reply:
Ohh okay, done🥳